new-client/prebuild.js

```diff
@@ -0,0 +1,59 @@
+const { execSync } = require("child_process");
+const fs = require("fs");
+
+const envLocalFile = ".env.local";
```

Comment:
Can you please add some defaults to `.env`, so admins can figure out the key names without looking inside the code? I guess that we can set our own values in `.env.local`, but defaults would be nice too.

Reply:
Yes, I've now added defaults. Also moved the npm vars to .env as these never change in prebuild.js